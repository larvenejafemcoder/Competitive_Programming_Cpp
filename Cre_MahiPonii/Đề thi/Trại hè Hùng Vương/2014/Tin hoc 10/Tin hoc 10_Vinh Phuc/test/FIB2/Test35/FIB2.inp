100
36 19495301
38 28296882
27 143062
6 10
42 336231892
20 291
38 52766359
12 203
30 344358
14 416
32 3519505
25 79983
13 22
11 82
26 110654
23 28019
28 99216
8 15
10 33
19 6186
13 277
5 7
34 3635779
43 90571811
36 3368013
39 92016162
17 2451
38 2094062
21 15534
41 112324513
44 204482253
28 329612
45 1245268392
10 18
25 88353
44 296776779
29 726286
28 118697
11 137
22 14411
15 733
34 313994
41 242769971
19 4267
3 3
37 509053
39 11263168
10 14
10 5
23 29859
26 176449
40 133169224
19 6375
20 10915
19 2801
10 10
38 52136660
37 19409225
30 633363
30 153551
35 6120881
2 2
5 7
31 1684418
42 90119316
43 410292946
28 263896
43 399342296
23 14119
6 12
21 631
4 1
42 354659717
26 85680
12 156
22 23511
7 12
2 2
30 721072
40 45149416
0 1
44 47544743
41 229391359
40 77160237
27 248968
35 3854531
35 8649596
43 413893729
44 801238147
41 8368434
26 156228
40 98268631
19 4436
21 7667
31 158801
42 161878413
10 25
24 66134
30 921132
18 749
