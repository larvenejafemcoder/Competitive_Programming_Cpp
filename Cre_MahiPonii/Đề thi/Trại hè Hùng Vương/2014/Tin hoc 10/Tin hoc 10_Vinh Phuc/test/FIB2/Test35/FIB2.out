7446542
10808447
54645
4
128429155
111
20154956
78
131533
159
1344331
30551
9
31
42266
10702
37897
6
13
2363
106
3
1388744
34595353
1286467
35147046
936
799861
5934
42904146
78105271
125901
475650201
7
33748
113358643
277417
45338
52
5505
280
119935
92729878
1630
1
194441
4302147
5
2
11405
67398
50866117
2435
4169
1070
4
19914432
7413664
241923
58651
2337969
1
3
643391
34422516
156717960
100799
152535184
5393
5
241
1
135467958
32727
60
8981
5
1
275425
17245542
1
18160476
87619703
29472588
95097
1472300
3303852
158093337
306045739
3196457
59674
37535277
1695
2929
60657
61832052
10
25261
351841
286
